cafebabel/articles/tags/views.py

```diff
@@ -36,3 +37,16 @@ def detail(slug):
     articles = Article.objects(tags__in=[tag], status='published')
     return render_template('articles/tags/detail.html', tag=tag,
                            articles=articles)
+
+
+@tags.route('/<slug>/edit/', methods=['get', 'post'])
+@editor_required
+def edit(slug):
+    tag = Tag.objects.get_or_404(slug=slug, language=current_language())
```

Comment:
I'd like to attract your attention on this @davidbgk.
This solves the issue when there are [twice the same slug in a different language](https://github.com/cafebabel/cafebabel.com/pull/153/files#diff-0896e0b2368aabe52dcdedab894926a3R106).

However this means that changing the language of the current slug would not make it available when redirecting after save.
Nevertheless, I understood that the expected behavior was to browse/work in the language version that you would be reading/editing.

It may be interesting to exchange about this.

Reply:
If I understand correctly [latest comment](https://github.com/cafebabel/cafebabel.com/issues/30#issuecomment-358245145) from @cafebabel-webmaster we should not be able to change the name, slug of language for a given tag anyway.

Reply:
OK, now I understand the problem. Thanks for the fix.